oidc-ui/src/App.css

```
@@ -0,0 +1,313 @@
+@tailwind components;
+@tailwind utilities;
+
+
+*,
+body,
+input,
+textarea,
+button,
+select {
+  font-family:
+    'Montserrat',
```

Stylelint (17.11.0)

[error] Expected no quotes around "Montserrat" (font-family-name-quotes)

(font-family-name-quotes)